src/components/DpDatetimePicker/DpDatetimePicker.stories.mdx

```diff
@@ -0,0 +1,25 @@
+import { Meta } from '@storybook/addon-docs'
+import DpDatetimePicker from './'
+
+<Meta
+    title="Components/DatetimePicker"
+    component={DpDatetimePicker}
+/>
+
+# DatetimePicker
+
+The DatetimePicker component allows users to enter a `date & time` either by typing or by selecting a `date & time`
+from the calendar.
+
+Is used to allow the user to select a date and time, and to display that date and time in the specified format.
```

Comment:
I think this sentence is redundant with the first sentence (The DatetimePicker component...".

```suggestion
```